### ISSUE TITLE
Install required libs

### ISSUE THREAD

**Issue by brolund** [liskin/hybrid-screenclone#2]
I had to execute:

sudo apt-get install libxcursor-dev libxdamage-dev libxinerama-dev libxtst-dev

before being able to run make, but after that it works like a charm.

**Comment by liskin** [comment]
You're not suggesting the makefile should do that for you, are you? :-)

**Comment by brolund** [comment]
Not at all, I just thought it might be good info to put someplace, perhaps in the readme? :-) 

Didn't have the energy to make a fork and a pull request only for that. ;-)
